lib/hotel_controller.rb

```diff
@@ -1,24 +1,44 @@
 module Hotel
-  class HotelController
-    # Wave 1
-    def rooms
-      # You might want to replace this method with an attr_reader
-      return []
-    end
+  class HotelController 
 
-    def reserve_room(start_date, end_date)
-      # start_date and end_date should be instances of class Date
-      return Reservation.new(start_date, end_date, nil)
-    end
+    attr_accessor :hotel_rooms, :reservations
+    attr_reader 
 
-    def reservations(date)
-      return []
+    def initialize()
```

Comment:
This constructor looks great. Make sure to include a test that check that the class is set up correctly.